docs/openebs-susecaas.yaml

```diff
@@ -0,0 +1,421 @@
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
```

Comment:
name nit : openebs-operator-susecaas.yaml  ?

Reply:
@prateekpandey14 I have change the file name